exercises/phone-number/example.rs

```diff
@@ -18,10 +25,14 @@ pub fn area_code(s: &str) -> Option<String> {
 }
 
 pub fn pretty_print(s: &str) -> String {
```

Comment:
since `pretty_print` is no longer being tested, you can and should now remove this function

You also can/should remove area_code